packages/react-components/react-tags/stories/TagGroup/TagGroupBestPractices.md

```diff
@@ -2,4 +2,8 @@
 
 ### Do
 
+- When using `TagGroup` with `Combobox` to display multiple selected values:
+  - Set the `listbox` role for `TagGroup` and the `option` role for each `Tag`.
+  - If using `TagButton`, set the `option` role for the content and make the dismiss button not focusable. When content is focused, Enter/Space should invoke the primary action, and Backspace/Delete remove the tag.
```

Comment:
we might eventually want more guidance here -- there's some stuff I didn't get into b/c of time like `aria-posinset`, but I think this is a good summary for now 👍